src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -440,4 +439,63 @@ public String convert(final String name) {
 		});
 	}
 	
+	@Override
+	public List<String> onTabComplete(CommandSender sender, Command cmd, String s, String[] args) {
+		File file = new File(Skript.getInstance().getDataFolder(), "scripts");

```

Comment:
This should probably use ScriptLoader to get the scripts see ExprScripts)

Reply:
That what used to get the new script file.

Reply:
What I mean is that instead of getting the files through here you should be doing it through ScriptLoader.
e.g.
ScriptLoader#getLoadedFiles
ScriptLoader#getDisabledFiles